picard/ui/mainwindow.py

```diff
@@ -944,6 +948,10 @@ def update_selection(self, objects=None):
                         }
                     self.set_statusbar_message(msg, mparms, echo=None,
                                                history=None)
+            elif isinstance(obj, Album):
+                obj.update_metadata_images()
```

Comment:
I didn't find any way to call an album's update_metadata_images when the images are changed in any track/file related to that album. Do you have any suggestion?